README.md

```diff
@@ -0,0 +1,72 @@
+# Damascene
+
+GPU gPb image segmentation.
+
+Basic install task list on ubuntu 16.04:
+
+ * Install cuda 8.0
+ * Install gcc-5
+ * Install ACML
+ * Run cmake and build libdamascene.so
+ * Build the python bindings
+ * ... start GPU segmenting
+
+### Install CUDA
+
+This assumes you have installed the proprietary nvidia drivers for ubuntu.  (although I think the cuda intall process may take care of that)
+
+```
+wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/cuda-repo-ubuntu1604_8.0.61-1_amd64.deb
```

Comment:
Is that 16.04 specific?

Reply:
Yes it is.
There is a 14.04 version available too.
https://developer.nvidia.com/cuda-downloads

Reply:
Can you maybe make a note of that?